software/spmd/pc_profiler/main.c

```diff
@@ -0,0 +1,90 @@
+// Program to print 10 fibonacci numbers; A sample program that will get profiled
+
+#include "bsg_manycore.h"
+#include "bsg_set_tile_x_y.h"
+
+// Histogram location?
+// 1. Can histogram base address be accessed directly from symbol table?
```

Comment:
I would think so.

Reply:
Yup, found a way to do that. Reflected in the latest commit.